include/SSVOpenHexagon/Utils/Utils.hpp

```diff
@@ -43,19 +43,39 @@ inline void uppercasify(std::string& s)
     return s;
 }
 
-inline float getFontHeight(sf::Text& font)
+[[nodiscard, gnu::pure]] inline float getFontHeight(sf::Text& font)
```

Comment:
```suggestion
[[nodiscard]] inline float getFontHeight(sf::Text& font)
```

`gnu::pure` can be a bit problematic with references involved. I am not sure it can be applied correctly here, so please remove it.